source/_components/cast.markdown

```diff
@@ -30,9 +30,20 @@ For example, Cast devices can only be discovered if they are on the same subnet
 cast:
   media_player:
   - host: 192.168.1.10
+  - port: 8009
 ```
 
 Configuration variables:
 
 - **host** (*Optional*): Use only if you don't want to scan for devices.
+- **port** (*Optional*): Specify chromecast port number - useful to define Chromecast Audio groups
 - **ignore_cec** (*Optional*) A list of Chromecasts that should ignore CEC data for determining the active input. [See the upstream documentation for more information.](https://github.com/balloob/pychromecast#ignoring-cec-data)
+
+### A note on Chromecast Audio groups
+Chromecast Audio groups are added as long as [the discovery component]({{site_root}}/components/discovery/) is used to discover your devices. If for some reason you wish to define your devices manually (i.e.: when running HA in another network), these groups can be manually added by specifying the group master's IP and the group's port.
+These values can be discovered by running netdisco in your local network, i.e.:
+```
+pip install netdisco
```

Comment:
It's unclear whether this needs to be done inside the Python venv, or outside of it. Please update to make it clear. Similarly how would this be done on Hass.io?